src/common/helpers/classname.ts

```diff
@@ -0,0 +1 @@
+export const cnc = (...names) => names.filter(n => n).join(' ');
```

Comment:
Add a comment indicating what is the purpose of _cnc_ and include a sample, I think @fjcalzado you are already added a comment in jfk.

We should consider this and other helper functions that we use in several projects publish them as a library in npm